Cmake structure (#364)
cda-tum/mqt-core: CMakeLists.txt

include(cmake/PackageAddTest.cmake)
include(cmake/Cache.cmake)

option(BUILD_MQT_CORE_BINDINGS "Build the MQT Core Python bindings" OFF)
if(BUILD_MQT_CORE_BINDINGS)

>   # ensure that the BINDINGS option is set in the MQT Core library
>   set(BINDINGS
>       ON
>       CACHE BOOL "Build the MQT Core Python bindings" FORCE)

Reviewer: ```suggestion
  # ensure that the BINDINGS option is set
  set(BINDINGS
      ON
      CACHE BOOL "Enable settings related to Python bindings" FORCE)
```
